cmd/shell.go

```diff
@@ -48,68 +48,76 @@ func (m mysqlLogger) Print(v ...interface{}) {
 	m.AuditErrf("[mysql] %s", fmt.Sprint(v...))
 }
 
+// grpcLogger implements the grpclog.LoggerV2 interface.
 type grpcLogger struct {
 	blog.Logger
 }
 
-// V returns true if the verbosity level l is less than the verbosity we want to
-// log at.
-func (log grpcLogger) V(l int) bool {
-	return l < 0
-}
-
+// Ensure that fatal logs exit, because we don't use the either the gRPC default
+// logger nor the stdlib default logger, both of which would call os.Exit(1) for
```

Comment:
```suggestion
// Ensure that fatal logs exit, because we neither the gRPC default
// logger nor the stdlib default logger, both of which would call os.Exit(1) for
```

Reply:
Done!